evap/contributor/forms.py

```diff
@@ -44,23 +44,26 @@ def clean(self):
 
         vote_start_datetime = self.cleaned_data.get('vote_start_datetime')
         vote_end_date = self.cleaned_data.get('vote_end_date')
+
+        # IMPORTANT: when removing following feature (auto-set a vote_start_datetime of the past to the future),
+        # make sure to alter the according test ("start date in the past -> form invalid?")!
+        # Location of test: evap/staff/tests/test_forms.py
+        #                   EvaluationFormTests.test_contributor_evaluation_form_date_validation(self):
+        if vote_start_datetime and vote_start_datetime < datetime.now():
+            self.cleaned_data['vote_start_datetime'] = round_datetime_up(round_to=5 * 60)
+
         if vote_start_datetime and vote_end_date:
             if vote_start_datetime.date() > vote_end_date:
                 self.add_error("vote_start_datetime", "")
                 self.add_error("vote_end_date", _("The first day of evaluation must be before the last one."))
 
-    def clean_vote_start_datetime(self):
```

Comment:
Have you thought about putting the cleaning logic for the vote start date in here? I mean, instead of throwing the error, you could add the fix here and just use that. Then, the form's clean function would not have to deal with this, at all.

Reply:
👍